lib/muffin_man/sp_api_client.rb

```diff
@@ -108,7 +108,7 @@ def request_lwa_access_token
           "Content-Type" => "application/x-www-form-urlencoded;charset=UTF-8"
         }
       )
-      raise SpApiAuthError, response if response.failure?
+      raise SpApiAuthError, response unless response.success?
```

Comment:
how about this?
```
if response.success?
  raise SpApiAuthError
elsif response['error'].present?
  raise SpApiAuthError, response['error_description']
end
```

can you check once if that will work in every condition? And also can we have test case for this?